### PR TITLE
[8.0] Add gs1 serial numbers on packages for logistics

### PR DESCRIPTION
**Pull request by rousseldenis**
Depends on PR #https://github.com/OCA/partner-contact/pull/298

### CODE REVIEW

File: stock_packaging_serial/static/description/icon.svg [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (lmignon) [inline stock_packaging_serial/static/description/icon.svg]: PLZ remove this file

File: stock_packaging_serial_sscc/static/description/icon.svg [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (lmignon) [inline stock_packaging_serial_sscc/static/description/icon.svg:1]: plz remove this file

**Comment by lmignon** [comment]
@rousseldenis Can you add unittests PLZ?

**Comment by rousseldenis** [comment]
@lmignon Ok, I'll do it, it should avoid fixing...

**Review by lmignon — CHANGES_REQUESTED** [review]

**Comment by rousseldenis** [comment]
@lmignon Done! But in maintainer-tools template we find both files (icon.svg and icon.png...)

**Review by lmignon — CHANGES_REQUESTED** [review]
Can you take a look to the pep8 error in travis?
https://travis-ci.org/OCA/stock-logistics-tracking/jobs/164057882#L430

**Pull request closed without merging**